Makefile

```diff
@@ -5,10 +5,9 @@ build:
 build-sequential:
 	@yarn turbo run --concurrency 1 build
 
-# We use the system Python due to this bug: https://github.com/astral-sh/python-build-standalone/issues/758
 venv-setup:
 	@[ -f .venv/bin/python3 ] || \
-		uv venv --python-preference only-system --python 3.10 --seed .venv || \
+		uv venv --python-preference only-managed --python 3.10 --seed .venv || \
 		python3 -m venv --upgrade-deps .venv
```

Comment:
Should we bite the bullet and mandate uv for setting up the venv and installing deps? I'm fine if we want to save that for a future diff, but let's consider filing an issue if so.

Reply:
I vote separate issue since this PR is so old.

Reply:
https://github.com/PrairieLearn/PrairieLearn/issues/13564